Game/Source/GDKTestGyms/Public/TestGymsCharacterMovementComp.h

```diff
@@ -23,7 +23,7 @@ class GDKTESTGYMS_API UTestGymsCharacterMovementComp : public UCharacterMovement
 	void ClientAuthServerMove_Implementation(const FVector_NetQuantize100& ClientLocation, const FVector_NetQuantize10& ClientVelocity, const uint32 PackedPitchYaw);
 	
 	bool bClientAuthMovement = false;
-	int32 ClientMoveFrequency = 10; // Send client position updates to server at this frequency
+	int32 ClientMoveFrequency = 30; // Send client position updates to server at this frequency
```

Comment:
Guess this affects all many of our tests. What was the purpose here?

Reply:
This is only used for client-auth movement rpcs, which are only used in the new disco scenario.

Reply:
And purpose was to make the scenario more "representative"